Makefile

```diff
@@ -78,7 +78,7 @@ build-sol:
 
 run:
 	mkdir -p deployments/starknet-devnet
-	poetry run starknet-devnet --lite-mode --seed 0 --dump-on exit --dump-path deployments/starknet-devnet/devnet.pkl --disable-rpc-request-validation --timeout 600
```

Comment:
is the disable-rpc-request-validation the cause of the cryptic rpc bugs?

Reply:
this option doesn't exist anymore

Reply:
but it was for skipping the strong type checking and there was mistmatches between api spec, devnet version and starknet py back in the days